ansible/roles/redpanda_broker/tasks/install-node-deps.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: include geerlingguy nodeexporter
+  ansible.builtin.include_role:
+    name: geerlingguy.node_exporter
+  tags:
+    - node_deps
+
+- name: fedora setup
+  when: ansible_os_family == 'RedHat'
+  tags:
+    - node_deps
+  block:
+    - name: delete unused repos
+      ansible.builtin.file:
+        name: "{{item}}"
+        state: absent
+      with_items:
+        - /etc/yum.repos.d/fedora-updates-modular.repo
+        - /etc/yum.repos.d/fedora-cisco-openh264.repo
```

Comment:
I know these were here before. but I don't know why we're doing this.

Reply:
Cargo cult rn, I haven't had a chance to test without some of this stuff.

Reply:
No problem. let's flag it for review later. this shouldn't holdup the pr.